Common/cpp/Registries/WorkletsCache.cpp

```diff
@@ -6,14 +6,6 @@ using namespace facebook;
 
 namespace reanimated {
 
-jsi::Value eval(jsi::Runtime &rt, const char *code) {
-  return rt.global().getPropertyAsFunction(rt, "eval").call(rt, code);
-}
-
-jsi::Function function(jsi::Runtime &rt, const std::string &code) {
-  return eval(rt, ("(" + code + ")").c_str()).getObject(rt).getFunction(rt);
-}
-
```

Comment:
This code was unused in the codebase. The eval was actually occurring later in the code, when doing `rt.evaluateJavascript`.